src/Transaction/BaseFirestoreTransactionRepository.spec.ts

```diff
@@ -366,7 +367,15 @@ describe('BaseFirestoreTransactionRepository', () => {
     it('should initialize nested subcollections', async () => {
       await bandRepository.runTransaction(async tran => {
         const pt = await tran.findById('red-hot-chili-peppers');
-        const album = await pt.albums.findById('stadium-arcadium');
+        const albumsRef = pt.albums;
+        if (!albumsRef) {
```

Comment:
I see a lot of this null checks. Is there a way to disable them for tests? If something that should not be null becomes null the tests will throw right away

Reply:
Agreed, I believe we can add tests to the `exclude` tsconfig option

Reply:
Removed the unnecessary if's :smile: